src/query/api/v1/handler/graphite/render_test.go

```diff
@@ -45,6 +46,24 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	testHandlerOptions options.HandlerOptions
+)
+
+func init() {
```

Comment:
nit: can this be an explicit test function taking in a `*t.Testing` and requiring no error on the NewFetchOptionsBuilder call? A little wary of `init` in tests having the potential to bleed into other tests, especially if we're not allocing large pools as part of the init